frontend/src/routes/(app)/report/[id]/[name]/+page.svelte

```diff
@@ -32,15 +30,15 @@
 		zenoClient
 			.addReportElement(data.report.id, {
 				type: ReportElementType.TEXT,
-				data: 'new element',
```

Comment:
Nit: if we do this numbered thing and don't count up, what's the point?

Reply:
Oops that was an accident was debugging.